build.ps1

```diff
@@ -5,7 +5,8 @@ param(
     [string]$root=$PSScriptRoot,
     [string]$runTests="YES",
     [string]$failBuildOnTest="YES",
-    [string]$slnFile="wilson.sln")
+    [string]$slnFile="wilson.sln",
+    [switch]$generateContractAssemblies)
```

Comment:
should we have a default here?

Reply:
The value is false by default when the parameter is not present. I'm checking if the switch is present with`
$generateContractAssemblies.IsPresent
` which _returns true if the parameter was specified on the command line, false otherwise._

Reply:
ok

Reply:
:shipit: